src/web/mjs/connectors/GuyaMoe.mjs

```diff
@@ -0,0 +1,12 @@
+import Guya from './templates/Guya.mjs';
+
+export default class GuyaMoe extends Guya {
```

Comment:
`GuyaMoe` was rejected due to only having a single manga, the website has no intrusive ads and there are also download links for each chapter